src/cdk/schematics/update-tool/component-resource-collector.ts

```diff
@@ -155,7 +155,7 @@ export class ComponentResourceCollector {
           return;
         }
 
-        const fileContent = this._fileSystem.read(templatePath);
+        const fileContent = stripBom(this._fileSystem.read(templatePath) || '');
```

Comment:
just fyi: need to be very careful here. It depends on the virtual tree and update recorder, whether they auto-insert the BOM offset or not. I just remember some issues there

Reply:
I figured it would be okay since it fixed the error and we've had that call for CSS files for a long time now.